sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -286,11 +288,14 @@ export class ServiceBusManagementClient extends ServiceClient {
    * @param queueName
    *
    */
-  async getNamespaceProperties(): Promise<NamespacePropertiesResponse> {
+  async getNamespaceProperties(
+    operationOptions?: OperationOptions
```

Comment:
The `@param` docs for `operationOptions` needs to be added to all the methods in this file

Reply:
Added